app/Models/QueryBuilders/SearchFormQueryBuilder.php

```diff
@@ -136,6 +170,26 @@ public function buildQuery(): array
             ];
         }
 
+        if ((isset($this->startDate) && !empty($this->startDate)) || (isset($this->endDate) && !empty($this->endDate))) {
```

Comment:
I think one of isset and empty functions is redundant.

Reply:
Nope, we need to provide option to either search from starting date or ending date or both or none at all.